### PR TITLE
Up TarikaWaldaAmid (Beta recension)

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
I updated LIT6936TarWaAmidBeta following my announcement in https://github.com/BetaMasaheft/Documentation/issues/2710. I made some minor change and a number of preliminary notes in LIT4723TarikaWaldaAmid, which I want to integrate correctly on another occasion.

### CODE REVIEW

File: 6001-7000/LIT6936TarWaAmidBeta.xml [hunk 1]
```diff
@@ -21,7 +21,25 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                    <witness corresp="BNFabb20" xml:id="A"/>
+                    <witness corresp="PetermannIINachtrag24" xml:id="D"/>
+                    <witness corresp="BNFet221" xml:id="E"/>
+                    <witness corresp="FSUor40" xml:id="H"/>
+                    <witness corresp="FSUor134"  xml:id="J"/>
+                    <witness corresp="BLorient814" xml:id="M">London, British Library, BL Oriental 814, which is indicated with M in 
+                        <bibl><ptr target="bm:Hoffmann2021Amid"/></bibl>, but with O in 
+                        <bibl><ptr target="bm:DeratSeignobos2015Moise"/></bibl>.</witness>
+                    <witness corresp="EMDA75" xml:id="N"/>
+                    <witness corresp="EMML21" xml:id="P">ʾAddis ʾAbabā, Manbara Lǝʿul Qəddus Mārqos, EMML 21, which is indicated with P
```
Comment (thea-m) [inline 6001-7000/LIT6936TarWaAmidBeta.xml:34]: `<bibl>` inside witness is used elsewhere as well and so probably correct, however, the zotero links will not be resolved as the entire content is rendered as a link to the manuscript record already.

File: 6001-7000/LIT6936TarWaAmidBeta.xml [hunk 2]
```diff
@@ -194,6 +250,362 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                                    <ab>
+                                        ወብሔር፡ ሳብዕሰ፡ ትሰመይ፡ በሮማይስጥ፡ ያሮስ፡ ወዘየዐውዳ።
+                                        ወይሰሐብ፡ መዓልቱ፡ ዘይነውኅ፡ ፲ወ፮ሰዓተ።
+                                    </ab>
+                                </div>
+                            </div>
+                            <div type="textpart" subtype="part" xml:id="Dimensions">
+                                <label>Dimensions of inhabited earth</label>
+                                <div type="textpart" subtype="part" xml:id="Width">
+                                    <label>Width of inhabited earth</label>
```
Comment (thea-m) [inline 6001-7000/LIT6936TarWaAmidBeta.xml:387]: My feeling would be to add "the" before "inhabited" (here and below)

File: 6001-7000/LIT6936TarWaAmidBeta.xml [hunk 3]
```diff
@@ -194,6 +250,362 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                                        ወይሰመይ፡ ፈለገ፡ ሐመልሚል።
+                                        ወይከውን፡ አንጻረ፡ ምድረ፡ ህንደኬ፡ እመንገለ፡ ምድረ፡ ህንደኬ፡ እምሥራቅ፡ ወይሰመይ፡ ጢራባይ።
+                                        የዐውድ፡ ቦቱ፡ ፴፻ምዕራፈ።
+                                        ውስቴታ፡ አድባር፡ ዐበይት፡ ወአፍላጋት፡ ብዙኅ። 
+                                        እምኔሃ፡ ይወፅእ፡ መረግድ፡ ቀይሕ፡ ወኅብረ፡ ሰማይ።
+                                        ወየዐውዳ፡ ለዛቲ፡ ደሴት፡ ፺ደሰያት፡ የኀድሩ፡ ውስቴቶሙ፡ ሰብእ፡ ወአህጉር፡ ብዙኅ።
+                                    </ab>
+                                </div>
+                                <div type="textpart" subtype="part" xml:id="GreenSea">
+                                    <label>The Green Sea</label>
```
Comment (thea-m) [inline 6001-7000/LIT6936TarWaAmidBeta.xml:587]: elsewhere no complete capitalisation, so probably better "sea"

**Review by thea-m — APPROVED** [review]
Great, thank you! Will you add the bibliography that is currently commented out later?

**Comment by CarstenHoffmannMarburg** [comment]
> Great, thank you! Will you add the bibliography that is currently commented out later?

I want to extend the bibliography later, but I am not sure, if all the secondary literature and editions are useful and needed. Some of the secondary literature mentioned here, are rather short notes on the dating of Tarika Walda Amid. Do you think, they are relevant and should be included?

File: 6001-7000/LIT6936TarWaAmidBeta.xml (hunk 1 above)
Comment (CarstenHoffmannMarburg) [reply 6001-7000/LIT6936TarWaAmidBeta.xml:34]: In this case, it might be better to write 'Derat/Seignobos 2015' explicitely and without markup.

**Comment by thea-m** [comment]
I think that any literature is useful, especially as you have collected it already and most of it is probably already present in zotero

File: 6001-7000/LIT6936TarWaAmidBeta.xml (hunk 1 above)
Comment (eu-genia) [reply 6001-7000/LIT6936TarWaAmidBeta.xml:33]: The citation will be resolved, just not the active link. But if each quoted bibliography item is listed in the bibliography for the record, as it should be, then it is not a problem.

@CarstenHoffmannMarburg note that I have updated the Zotero record which missed the names of the editors, the full title, the place, and the publisher. It is important that all Zotero items are as correct and complete as possible, too, unfortunately we have no possibility of consistent peer review there so everyone must be very careful, thank you.

**Comment by eu-genia** [comment]
@CarstenHoffmannMarburg can you check all the Zotero records you created that should clearly distinguish authors / editors / translators, for foreign languages provide the original title and transcription and translation, same pattern as for Ethiopic, always have place of publication, publisher, series if applicable etc. - all information that appears on the frontispiece. Since these are books you use you surely can see exactly what is there. I have fixed Derat-Seignobos 2015 but I notice that also KrackovskijFrantsouzoff2015Geography is incomplete and maybe some other, thank you!

**Comment by CarstenHoffmannMarburg** [comment]
> @CarstenHoffmannMarburg can you check all the Zotero records you created that should clearly distinguish authors / editors / translators, for foreign languages provide the original title and transcription and translation, same pattern as for Ethiopic, always have place of publication, publisher, series if applicable etc. - all information that appears on the frontispiece. Since these are books you use you surely can see exactly what is there. I have fixed Derat-Seignobos 2015 but I notice that also KrackovskijFrantsouzoff2015Geography is incomplete and maybe some other, thank you!

I do not remember how many and which Zotero records, I have created by myself. Is there a possibility to look it up?
I do not have any additional information on KrackovskijFrantsouzoff2015Geography, which I could add. The bibliographic informaition from my dissertation is the following: 
Kračkovski, Ignatij Julonovič: Iz ethiopskoj geografičeskoj literatury, in: Hristianskij Vostok 1 (1912), S. 127-145 = Atti della Imperiale Accademia di Pietroburgo, Bd. 1, sowie in verbesserter Ausgabe von Serguei Frantsouzoff in: Kračkovski, Ignatij Julonovič: Trudy po istorii i filologii hristianskovo Vostoka, Moskau: 2015, S. 257-258; 268-279; 290-300 u. 631.
Other information, I do not have.

**Pull request merged**